src/utils/plot_results.jl

```diff
@@ -54,7 +54,10 @@ function get_stacked_plot_data(res::OperationsProblemResults, variable::String;
 
     data_matrix = convert(Matrix, variable)
     labels = collect(names(variable))
-    legend = string.(labels)
+    legend = [names(variable)[1]]
```

Comment:
In the existing code it seems odd that `variable` is passed into the function and then immediately overwritten.

What does `names()` do?  That is a Base function to get the names exported by a module.